src/app/results/results.component.css

```diff
@@ -654,3 +639,23 @@ div.autocorrect{
     padding: 0px 2vw 12px;
   }
 }
+
+@media screen and (max-width: 320px) {
+  #search-options {
+    margin-left: 0.3%;
+  }
+  .feed.col-md-10.col-md-offset-1.col-sm-10.col-sm-offset-1.col-lg-10.col-lg-offset-1 {
```

Comment:
There is no need for this, the large screen pull automatically fixes this, please let it get merged